src/main/java/edu/illinois/cs/dt/tools/utility/iFixPlusPlugin.java

```diff
@@ -0,0 +1,744 @@
+    private String dtname;
+    private String subxmlFold;
+    private String rootFold;
+    private String module;
+    private String output;
+    private String slug;
+    private String tmpfile;
+    private String diffFieldsFold;
+    private String subdiffsFold;
+    private String reflectionFold;
+
+    private Set<String> diffFields_filtered = new HashSet<String> ();
+
+    @Override
+    public void execute(final MavenProject mavenProject) {
+        Configuration.config().properties().setProperty("statecapture.phase", "initial");
+
+        long startTime = System.currentTimeMillis();
+
+        // Currently there could two runners, one for JUnit 4 and one for JUnit 5
+        // If the maven project has both JUnit 4 and JUnit 5 tests, two runners will
+        // be returned
+        List<Runner> runners = RunnerFactory.allFrom(mavenProject);
+
+        if (runners.size() != 1) {
+            // HACK: Always force JUnit 4
+            boolean forceJUnit4 = true;
+            if (forceJUnit4) {
+                Runner nrunner = null;
+                for (Runner runner : runners) {
+                    if (runner.framework().toString() == "JUnit") {
+                        nrunner = runner;
+                        break;
+                    }
+                }
+                if (nrunner != null) {
+                    runners = new ArrayList<>(Arrays.asList(nrunner));
+                } else {
+                    String errorMsg;
+                    if (runners.size() == 0) {
+                        errorMsg =
+                            "Module is not using a supported test framework (probably not JUnit), " +
+                            "or there is no test.";
+                    } else {
+                        errorMsg = "dt.detector.forceJUnit4 is true but no JUnit 4 runners found. Perhaps the project only contains JUnit 5 tests.";
+                    }
+                    TestPluginPlugin.mojo().getLog().info(errorMsg);
+                    return;
+                }
+            } else {
+                String errorMsg;
+                if (runners.size() == 0) {
+                    errorMsg =
+                        "Module is not using a supported test framework (probably not JUnit), " +
+                        "or there is no test.";
+                } else {
+                    // more than one runner, currently is not supported.
+                    errorMsg =
+                        "This project contains both JUnit 4 and JUnit 5 tests, which currently"
+                        + " is not supported by iDFlakies";
+                }
+                TestPluginPlugin.mojo().getLog().info(errorMsg);
+                return;
+            }
+        }
+        final Runner runner = InstrumentingSmartRunner.fromRunner(runners.get(0));
+
+        replayPath = Paths.get(Configuration.config().getProperty("replay.path"));
+        replayPath2 = Paths.get(Configuration.config().getProperty("replay.path2"));
```

Comment:
Similar thing about the property name itself. In fact, should they all have the prefix "replay"? If they are more relevant for controlling testrunner's statecapture logic, maybe they should instead be prefix "statecapture"?